cirq/contrib/quimb/mps_simulator.py

```diff
@@ -415,7 +430,7 @@ def to_numpy(self) -> np.ndarray:
         """An alias for the state vector."""
         return self.state_vector()
 
-    def apply_op(self, op: 'cirq.Operation', prng: np.random.RandomState):
```

Comment:
This changes the API for this class - is there a use case for his method outside of `act_on`? If so, `apply_op` should be kept and `_act_on_fallback_` converted to:
```
def _act_on_fallback_(self, op: Any, allow_decompose: bool):
    return self.apply_op(op, self.prng)
```
Otherwise, since this is under `contrib` deprecation isn't necessary.